.github/workflows/1-release.yml

```diff
@@ -4,48 +4,90 @@ on:
     branches: [main]
 
 jobs:
-  release_action:
+  release_new_version:
```

Comment:
@mastacheata Didnt check the details but there needs to be some documentation on how to use this, right?

Reply:
The idea is this will take care of itself.
It creates a new release for every commit that is merged into main.
The version number is also auto-generated depending on the commit/PR subject (fix/feature/chore etc. and then BREAKING in All-caps if something is a breaking change)

It also triggers another action at the main repo where all the split-off-checks are combined into one and updates the include directives there.
not yet sure when/how to create a new version for the main repo, though.

Reply:
Nevertheless, it might be helpful if this is documented somewhere, right? At least in the case that it breaks at some point.